docs/source/reference/runners/parallel_for.rst

```diff
@@ -48,7 +48,7 @@ This example builds on the previous one and shows how to consume the iteration i
             print(message)
             return self.build_success_message(message=message)
 
-    chain = Chain(name="name", description="parallel for", runners=[ParallelFor(generator, MySkill(), parallelism=1)])
+    chain = Chain(name="name", description="parallel for", runners=[ParallelFor(generator, MySkill(), parallelism=5)])
```

Comment:
Should we introduce an env var with a default value equal to 5 `COUNCIL_DEFAULT_PARALLELISM` ?

Reply:
that parameter is directly accessible when building a Chain. I'm not sure an EnvVar would add much value here